docs/api/programme.md

```diff
@@ -0,0 +1,251 @@
+        "acronym": "LEIC",
+        "termSize": 6
+    },
+    "entities": [
+        {
+            "class": [ "offer" ],
+            "title": "LS Offer",
+            "rel": [ "/rel/programmeOffer" ],
+            "properties": {
+                "CourseId": 2,
+                "TermNumber": 3
+            },
+            "links" : [
+                { "rel": [ "self" ], "href": "/v0/programmes/LEIC/offers/1"}
+            ]
+        },
+        {
+            "class": [ "offer" ],
+            "title": "AED Offer",
+            "rel": [ "/rel/programmeOffer" ],
+            "properties": {
+                "CourseId": 5,
+                "TermNumber": 3
+            },
+            "links" : [
+                { "rel": [ "self" ], "href": "/v0/programmes/LEIC/offers/2"}
+            ]
+        },
+        {
+            "class": [ "offer" ],
+            "title": "POO Offer",
+            "rel": [ "/rel/programmeOffer" ],
+            "properties": {
+                "CourseId": 4,
+                "TermNumber": 1
+            },
+            "links" : [
+                { "rel": [ "self" ], "href": "/v0/programmes/LEIC/offers/3"}
+            ]
+        }
+    ],
+    "actions": [
+        {
+            "name": "edit-programme",
+            "title": "Edit Programme",
+            "method": "PUT",
+            "href": "/v0/programmes/1",
+            "type": "application/json",
+            "fields": [
+                { "name": "ProgrammeName", "type": "text"}, //the name may be NULL, you can edit it
+                { "name": "Acronym", "type": "text"},
+                { "name": "TermSize", "type": "number"}
+            ]
+        },
+                {
+            "name": "add-offer",
+            "title": "Add Offer",
+            "method": "POST",
+            "href": "/v0/programmes/1/offers",
+            "type": "application/json",
+            "fields": [
+                { "name": "CourseId", "type": "number"},
+                { "name": "CurricularTerm", "type": "number" },
+                { "name": "Optional", "type": "boolean"}
+            ]
+        }
+    ],
+    "links": [
+        { "rel": [ "self" ], "href": "/v0/programmes/1" },
+        { "rel": [ "up" ], "href": "/v0/programmes/" }
+    ]
+}
+```
+
+---------------------------------------------------------------------
+# Programme Collection
+A collection of the possible programmes.
+
+## Link Relations
+   * A programme representation
+    - must include a link to its context, using the self link relation.
+    - may include links to the details of a programme, using the item link relation.
+
+## Actions
+    * add: add's a programme
+
+## Example representation
+```json
+{
+    "class": [ "collection", "programme" ],
+    "entities": [
+        {
+            "class": [ "programme" ],
+            "rel": [ "item" ],
+            "properties": {
+                "id": 1,
+                "acronym": "LEIC",
+                "termSize": 6
```

Comment:
Term size is a detail that should be present, in my opinion, in the Program's full representation. The program's representation as a collection item should only contain essencial information such as the ID and Acronym.